src/assets/sass/website/containers/about.scss

```diff
@@ -0,0 +1,118 @@
+@import "../color-palette.scss";
+@import "../variables.scss";
+@import "../mixins.scss";
+@import "../shared/half-clipped-shape.scss";
+
+.about-wrapper {
+  background-color: $primary;
+  height: max-content;
+  margin-top: -$header-height;
+  min-height: calc(100vh - #{$footer-height});
+  @include is-mobile {
+    margin-top: -$header-height-mobile - 45;
+    min-height: calc(100vh - #{$footer-height-mobile});
+  }
+  @media (max-width:600px) {
+    margin-top: -75px;
+  }
+  .hero-wrapper{
+    @media (max-width:600px) {
+      padding-top: 150px;
+    }
+  }
+  .about-container{
+    .half-filled-box{
+      margin: 0!important;
+      margin-bottom: 100px!important;
+      @include is-large-desktop {
+        margin-bottom: 90px!important;
+      }
+      @include is-desktop {
+        margin-bottom: 80px!important;
+      }
+      @include is-tablet {
+        margin-bottom: 70px!important;
+      }
+      @include is-mobile {
+        margin-bottom: 50px!important;
+        margin-top: 100px!important;
+      }
+    }
+    &__heroContent{
+      width: 80%;
+      height: 100%;
+      margin: auto;
+      text-align: left;
+
+        h1{
+            color: $header-item;
+            user-select: none;
+            font-size: 2.5rem;
+            font-family: Segoe UI !important;
+            font-weight: bolder;
+            @media (max-width:1650px) {
```

Comment:
create a variable for 1650px at the top of the file or in variables.scss if u use it in other files